tsconfig.json

```diff
@@ -10,8 +11,9 @@
         "pretty": true,
         "emitDecoratorMetadata": true,
         "experimentalDecorators": true,
-        "strictNullChecks": true,
+        "strictNullChecks": false,
```

Comment:
We use true in Companies House unless there is a good reason for false

Reply:
reverted